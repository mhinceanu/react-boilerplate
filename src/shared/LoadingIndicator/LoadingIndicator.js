import React from 'react';
import './LoadingIndicator.scss';

const LoadingIndicator = () => {
  return <div className="bp-loader"></div>;
};

export default LoadingIndicator;
