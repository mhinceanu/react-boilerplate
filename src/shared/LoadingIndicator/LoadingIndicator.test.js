import React from 'react';
import { render } from '@testing-library/react';
import LoadingIndicator from './LoadingIndicator';

describe('<LoadingIndicator />', function () {
  const getComponent = () => {
    return <LoadingIndicator />;
  };

  it('should render <LoadingIndicator />', () => {
    const { container } = render(getComponent());

    expect(container).toMatchSnapshot();
  });
});
