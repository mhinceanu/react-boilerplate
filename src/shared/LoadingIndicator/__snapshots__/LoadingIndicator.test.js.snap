// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingIndicator /> should render <LoadingIndicator /> 1`] = `
<div>
  <div
    class="bp-loader"
  />
</div>
`;
